source/_includes/pup43_platforms_fedora.markdown

```diff
@@ -0,0 +1,7 @@
+We publish official packages and run automated testing on the following versions for Puppet 4.3:
```

Comment:
We should probably go back and change the 4.0 ones as well. Technically, we aren't creating the packages for Puppet, we're creating them for puppet-agent. It looks like the OS X includes are the most correct:

"We publish official `puppet-agent` packages for the following Mac OS X versions:" 

Puppet is totally left out of the sentence here, which I think is more correct. The runner up I think would be your other suggestion,

"We publish and test official puppet-agent packages containing Puppet 4.3 on the following versions of Fedora:"

Because of the word "containing", I think this clears up the fact that the package isn't for puppet, it's for AIO.